mwclient/client.py

```diff
@@ -440,13 +530,38 @@ def raw_api(self, action, http_method='POST', *args, **kwargs):
             raise errors.InvalidResponse(res)
 
     def raw_index(self, action, http_method='POST', *args, **kwargs):
-        """Sends a call to index.php rather than the API."""
+        """Sends a call to index.php rather than the API.
+
+        Args:
+            action (str): The MediaWiki API action to perform.
+            http_method (str): The HTTP method to use in the request.
+
+        Returns:
+            The API response.
+        """
         kwargs['action'] = action
         kwargs['maxlag'] = self.max_lag
         data = self._query_string(*args, **kwargs)
         return self.raw_call('index', data, http_method=http_method)
 
     def require(self, major, minor, revision=None, raise_error=True):
+        """Check whether the current wiki matches the required version.
+
+        Args:
+            major (int): The required major version.
+            minor (int): The required minor version.
+            revision (int): The required revision.
+            raise_error (bool): Whether to throw an error if the version of the current
+                wiki is below the required version. Defaults to `True`.
+
+        Returns:
+            `False` if the version of the current wiki is below the required version, else
+                `True`. If `raise_error=True` then nothing is returned.
```

Comment:
It also returns nothing if the site is not yet initialized and `raise_error` is `None` ( but not if `raise_error` is anything *other* than `None`, including `False`, which is odd). This whole method is *bizarre*. PR time!

Reply:
I've updated the `Returns:` section:

```
`False` if the version of the current wiki is below the required version, else
`True`. If either `raise_error=True` or the site is uninitialized and
`raise_error=None` then nothing is returned.
```